meshtastic/interfaces/ble/utils.py

```
@@ -1,12 +1,166 @@
 """Utility functions for BLE operations."""
 
 import asyncio
+import inspect
 import importlib
 import time
 from types import ModuleType
-from typing import Awaitable, Callable, TypeVar
+from typing import Any, Awaitable, Callable, TypeVar
```

Ruff (0.15.5)

[warning] Import from `collections.abc` instead: `Awaitable`, `Callable`

Import from `collections.abc`

(UP035)